src/main/resources/edu/cornell/kfs/coa/document/workflow/CuObjectCodeActivationGlobalMaintenanceDocument.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>OCAG</name>
+      <parent>COA</parent>
+      <label>Object Code Activation Global</label>
+      <authorizer>org.kuali.kfs.krad.workflow.authorizer.CfDocumentTypeAuthorizer</authorizer>
+      <docHandler>${kfs.url}/objectcodeactivationglobalmaintenance.do?methodToCall=docHandler</docHandler>
+      <active>true</active>
+      <routingVersion>1</routingVersion>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="OrganizationHierarchy" />
+          <role name="OrganizationHierarchy" nextNode="Chart" />
+          <role name="Chart" />
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <role name="OrganizationHierarchy">
+          <!--<ruleTemplate>KualiOrgReviewTemplate</ruleTemplate>-->
+          <qualifierResolverClass>org.kuali.kfs.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+        </role>
+        <role name="Chart">
+          <!--<ruleTemplate>KualiChartTemplate</ruleTemplate>-->
```

Comment:
Another commented-out element to remove.

Reply:
ok